requirements_all_ds.txt

```diff
@@ -45,3 +45,6 @@ pandas==1.3.4
 nzpy>=1.15
 nzalchemy
 python-arango==6.1.0
+pinotdb>=0.4.1
+httpx==0.23.0
+six==1.11.0
```

Comment:
```suggestion
```

Why install `httpx` here? It doesn't appear to be used anywhere in the query runner. Redash uses `requests` for HTTP operations.

`six` is already a dependency of other libraries so we don't need to pin it here.